apps/react-storybook/stories/chat/Chat.stories.tsx

```diff
@@ -390,3 +390,60 @@ export const Customization: Story = {
         );
     }
 }
+
+export const Formatting: Story = {
+    args: {
+        dayHeaderFormat: 'shortdate',
+        messageTimestampFormat: 'shorttime',
+    },
+    argTypes: {
+        dayHeaderFormat: {
+            control: 'select',
+            options: ['shortdate', 'shorttime', 'yyyy-mm-dd hh:mm:ss', 'hh:mm:ss', 'longdate', 'longtime', 'none'],
+            mapping: {
+                ['shortdate']: 'shortdate',
+                ['shorttime']: 'shorttime',
+                ['yyyy-mm-dd hh:mm:ss']: 'yyyy-mm-dd hh:mm:ss',
+                ['hh:mm:ss']: 'hh:mm:ss',
+                ['longdate']: 'longdate',
+                ['longtime']: 'longtime',
+                ['none']: null,
+            },
+            defaultValue: 'shorttime',
+        },
+        messageTimestampFormat: {
+            control: 'select',
+            options: ['shortdate', 'shorttime', 'yyyy-mm-dd hh:mm:ss', 'hh:mm:ss', 'longdate', 'longtime', 'none'],
```

Comment:
I think that options and mapping are very similar beetween dayHeaderFormat & messageTimestampFormat. Can we avoid duplication?

Reply:
d698f0da587e7ca96a22174637de190efd7de254